modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/RootQuery.java

```diff
@@ -142,16 +139,10 @@ public RootQuery(
 
         FrameworkConfig frameworkCfg = qryCtx != null ? qryCtx.unwrap(FrameworkConfig.class) : null;
 
-        if (frameworkCfg == null)
```

Comment:
This change is unclear. We still have static `static final FrameworkConfig FRAMEWORK_CONFIG`. In general, the idea to make it non-statis, configurable look good. But do we need this in current ticket?

Reply:
Fixed by another ticket